cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -199,7 +198,11 @@ public static EntityTopicOperator fromCrd(Kafka kafkaAssembly) {
                         Labels.fromResource(kafkaAssembly).withKind(kafkaAssembly.getKind()));
 
                 result.setOwnerReference(kafkaAssembly);
-                result.setImage(topicOperatorSpec.getImage());
+                String image = topicOperatorSpec.getImage();
+                if (image == null) {
+                    image = System.getenv().getOrDefault("STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE", "strimzi/operator:latest");
```

Comment:
Do we relay need to have `strimzi/operator:latest` on 10+ different places across the code base?

Reply:
Yeah.... I guess we don't have another way because we can't have it in models as constants. This will cause them to be set by default. or maybe you can recommend something?

Reply:
I created `Environment` in one of my opened PRs. This class should contains thinks like this to avoid specify it on10+ different places in code.